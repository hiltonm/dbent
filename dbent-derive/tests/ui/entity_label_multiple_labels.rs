use dbent::prelude::*;

#[derive(Label)]
struct Test {
    id: Key<Int>,
    #[label] data1: String,
    #[label] data2: String,
}

fn main() {}
