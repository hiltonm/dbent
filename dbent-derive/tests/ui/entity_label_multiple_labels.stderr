error: #[derive(Label)] needs to have 1 field marked with #[label]
 --> tests/ui/entity_label_multiple_labels.rs:4:13
  |
4 |   struct Test {
  |  _____________^
5 | |     id: Key<Int>,
6 | |     #[label] data1: String,
7 | |     #[label] data2: String,
8 | | }
  | |_^
